### PR TITLE
Fix edge case: rapid re-init and destroy

### PR DESCRIPTION
**Pull request by jnu**
**Scenario**
`powerTip` is instantiated on an element that already has a `powerTip`, there is only one tooltip on the page, and `destroy` is called on that tooltip before it has a chance to show fully.

**Problem**
`session.tooltips` is `null` when `$.powerTip.destroy` tries to call `remove` on it.

**Solution**
Don't call `remove` when `session.tooltips` is `null`. Don't need to do anything there, since clearly there's nothing left to clean up.

There might be more stuff to fix if `powerTip` is instantiated multiple times on the same element, but this is the only issue I've seen with it.

### CODE REVIEW

File: test/unit/core.js [hunk 1]
```diff
@@ -254,6 +254,44 @@ $(function() {
 		strictEqual(session.currentX, 1, 'document event removed');
 	});
 
+	test('API destroy method will not fail when rapidly created, shown, and destroyed', function() {
```
Comment (nrabinowitz) [inline test/unit/core.js:297]: I'm confused - isn't this test in #2?

Comment (jnu) [inline test/unit/core.js:297]: I developed them at once and then tried to break them into separate PRs, I guess I partial committed wrong. The test belongs here though ..

Comment (nrabinowitz) [inline test/unit/core.js:297]: Makes sense, but you'll need to fix the merge conflict...

**Comment by nrabinowitz** [comment]
LGTM
